.github/actions/verify-token/package.json

```diff
@@ -1,8 +1,13 @@
 {
   "name": "verify-token",
   "version": "1.0.0",
-  "description": "",
-  "main": "lib/main.js",
+  "description": "Verify the SLSA token and generate a SLSA predicate",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
```

Comment:
out of curiosity: where do we instruct ncc to "complile" the files into `dist/dist` instead of `dist`?

Reply:
Right now I just copy the files manually from `dist` to `dist/dist` in the `Makefile`. Probably I could specify it with `ncc -o dist/dist` though so I'll try that.